bundles/org.openhab.binding.bambulab/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>5.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.bambulab</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: BambuLab Binding</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>pl.grzeslowski</groupId>
+      <artifactId>JBambuAPI</artifactId>
+      <version>1.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.paho</groupId>
+      <artifactId>org.eclipse.paho.client.mqttv3</artifactId>
```

Comment:
Maybe there is a specific reason or maybe you are not aware.
But for most mqtt enabled bindings, the feature / substrucutre is used. ou could look at any of the mqtt subbindings like `mqtt.awtrixlight`

Reply:
This dep comes from JBambuAPI library